ouroboros-consensus/src/Ouroboros/Consensus/Storage/ChainDB/Impl/LgrDB.hs

```diff
@@ -293,21 +295,31 @@ currentPoint = castPoint
              . ledgerState
              . LedgerDB.ledgerDbCurrent
 
+-- | Returns /the oldes/ ledger state currently in LedgerDB. This is just
+-- an alias to ledgerDbAnchor to give more semantic meaning
+getOldestLS :: LedgerDB' l -> ExtLedgerState l
```

Comment:
I don't think we use the `LS` abbreviation anywhere else (right?), so let's continue spelling it out for now.

Also, since `getCurrent` from this module is an `STM` action, I suggest renaming this to `oldestLedgerSnapshot`. (That matches the scheme used by `currentPoint` eg.)